django/contrib/humanize/templatetags/humanize.py

```diff
@@ -24,12 +24,14 @@
 def ordinal(value):
     """
     Convert an integer to its ordinal as a string. 1 is '1st', 2 is '2nd',
-    3 is '3rd', etc. Works for any integer.
+    3 is '3rd', etc. Works for any non-negative integer.
     """
     try:
         value = int(value)
     except (TypeError, ValueError):
         return value
+    if value < 0:
```

Comment:
```suggestion
    if value <= 0:
```
I don't think we should support `0th` as it doesn't really make sense